src/transformers/training_args.py

```diff
@@ -58,8 +58,26 @@ class TrainingArguments:
         default=False, metadata={"help": "Run evaluation during training at each logging step."},
     )
 
-    per_gpu_train_batch_size: int = field(default=8, metadata={"help": "Batch size per GPU/CPU for training."})
-    per_gpu_eval_batch_size: int = field(default=8, metadata={"help": "Batch size per GPU/CPU for evaluation."})
+    per_device_train_batch_size: Optional[int] = field(
+        default=None, metadata={"help": "Batch size per GPU/TPU core/CPU for training."}
+    )
+    per_device_eval_batch_size: Optional[int] = field(
+        default=None, metadata={"help": "Batch size per GPU/TPU core/CPU for evaluation."}
```

Comment:
those should have the same defaults as below no?

Reply:
If it's not set as `optional` (and therefore has an integer value instead of a `None` value, then there's no way of identifying if they have been given or not